neo/Wallets/AssetDescriptor.cs

```diff
@@ -19,7 +19,7 @@ public AssetDescriptor(UInt160 asset_id)
                 sb.EmitAppCall(asset_id, "name");
                 script = sb.ToArray();
             }
-            ApplicationEngine engine = ApplicationEngine.Run(script);
+            ApplicationEngine engine = ApplicationEngine.Run(script, extraGAS: 10_000_000);
```

Comment:
what's this value?

Reply:
We can adjust to a better one

Reply:
This seems arbitrary. Would setting test mode be a better approach?

Reply:
`name()` may contain an infinite loop, and `testMode` can cause the client to fail to jump out of an infinite loop.